src/test/java/com/github/redouane59/twitter/nrt/ITwitterClientV2Test.java

```diff
@@ -33,7 +35,7 @@
 public class ITwitterClientV2Test {
 
   private static TwitterClient twitterClient;
-  private        String        userId = "1307302673318895621";
+  private String userId = "1307302673318895621";
```

Comment:
are you sure you used the project style ? This should not be modified

Reply:
Honestly, using vscode, i could not use your GoogleStyle project style. Will try once again tonight.

Reply:
On my side I'm using Intellij but I don't know how to manage code styles in vscode :/

Reply:
I've switched to Intellij. There's nothing like the formatting features of Intellij in vscode.